golem-service-base/src/config.rs

```diff
@@ -58,3 +59,77 @@ impl Default for WorkerExecutorClientCacheConfig {
         }
     }
 }
+
+#[allow(clippy::large_enum_variant)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(tag = "type", content = "config")]
+pub enum BlobStorageConfig {
```

Comment:
No changes here, just moved to the service-base crate

Reply:
Just to give you a history, `golem-service-base` is probably a carry forward of a very early version of golem, and personally I was wishing everything that 's common just stays within golem-common. This moving may hinder the avoidance of `golem-service-base` in near future. Is this mandatory to have it here? 

Thanks for the comments though, that it clarifies the context.

Reply:
It's good to move this here for now.